Makefile

```diff
@@ -259,6 +264,12 @@ clang_tidy_diff:
 		| grep -E "\.h\$$|\.cpp\$$"            \
 		| xargs --no-run-if-empty ruby tools/run-clang-tidy.rb $(CMAKE_TOOLS_BUILD_DIR)
 
+check_os_version_valid:
+	@echo ""
+	@echo "🔨 Checking os version is valid ☑"
+	@echo ""
+	python3 tools/check_version.py $(FIRMWARE_VERSION) --standards semver mcuboot
```

Comment:
I'm not sure we need this check added to the Makefile. It is highly specific and not useful on a day to day basis.

I would prefer we call the script directly, like we do for `check fs content`